android/engine/src/main/java/org/smartregister/fhircore/engine/ui/questionnaire/QuestionnaireViewModel.kt

```diff
@@ -132,6 +131,28 @@ constructor(
     return questionnaireConfig
   }
 
+  suspend fun getQuestionnaireConfigPair(
+    context: Context,
+    formName: String,
+    type: QuestionnaireType
+  ): Pair<QuestionnaireConfig, Questionnaire> {
+    return try {
+      val config = getQuestionnaireConfig(formName, context)
+      val questionnaire = loadQuestionnaire(config.identifier, type)!!
+      Pair(config, questionnaire)
+    } catch (e: Exception) {
```

Comment:
can we catch something more specific here?

Reply:
My intention was to catch two exceptions `NullPointerException` and `NoSuchElementException`. From this [thread](https://discuss.kotlinlang.org/t/does-kotlin-have-multi-catch/486/24) it seems kotlin doesn't that good of support for multiple-catch, and just opted for the generic